src/objective-c/GRPCClient/private/GRPCHost.m

```diff
@@ -106,13 +106,24 @@ - (grpc_call *)unmanagedCallWithPath:(NSString *)path completionQueue:(GRPCCompl
 - (GRPCChannel *)channel {
   // Create it lazily, because we don't want to open a connection just because someone is
   // configuring a host.
+
   if (!_channel) {
+    NSMutableDictionary *args = [NSMutableDictionary dictionary];
+    NSString *userAgentPrefix = [[GRPCCall userAgentPrefix] copy];
```

Comment:
Ah.  No we don't.  We don't have a need since the dictionary and string aren't being kept past the channel creation call. I'll remove.